src/sensors/src/Accelerometer.cpp

```diff
@@ -139,6 +138,20 @@ SensorType Accelerometer::getSensorType() const
     return ACCELEROMETER;
 }
 
+bool Accelerometer::getLinkSensorTransform(Transform& link_H_sensor) const
+{
+    if( this->pimpl->parent_link_index < 0)
```

Comment:
Are we sure that we want to check this in the getter ? `parent_link_index` is not used at all in this method, so if a user creates a new Accelerometer, even if I don't set the parent link index I would expect to be able to set/get the link/sensor transform.